modules/rpc/thriftrw-plugin-thriftsync/README.md

```diff
@@ -3,17 +3,45 @@
  *.thrift file. With the use of thriftsync plugin, a user who needs to build a service should be able
  to auto generate the code and write service specific logic without worrying about underlying platform.
 
+## Usage
+Run thriftsync on any thrift file update to sync your handler code with the methods in the thrift file.
+To run thriftsync, you need to install thriftsync from vendor and run thriftrw code generation command.
+
+*Install thriftsync from vendor:*
+`go install ./vendor/go.uber.org/fx/modules/rpc/thriftrw-plugin-thriftsync`
+
+*Run thriftrw code genration with thriftsync*
+`thriftrw --plugin="thriftsync --yarpc-server=<Import path of the yarpc servicenameserver>" <thrift filepath>`
+
+Update your makefile with following lines and simply run `make thriftsync`
+*Update makefile*
+```
+deps:
+	@echo "Installing thriftrw..."
+	$(ECHO_V)go install ./vendor/go.uber.org/thriftrw
```

Comment:
i'd remove the echo v stuff -- we use it internally but for examples it just makes it less clear what the important stuff is